app/forms/provider_interface/offer_wizard.rb

```diff
@@ -62,9 +85,9 @@ def next_step(step = current_step)
 
     delegate :previous_step, to: :wizard_path_history
 
-    def configure_additional_conditions(conditions)
-      conditions.each_with_index do |condition, index|
-        instance_variable_set "@further_condition_#{index + 1}", condition
+    def conditions_models
```

Comment:
I find this method name confusing as this is not a model and also, why is it important?

Reply:
It is a list of models, which are passed to the fields_for in the view.

This is done because:
1. It commonises the validation of condition length
2. It allows the params coming from the form to be correctly indexed (which is why `id` is needed) which hooks up to formbuilder's error summary nicely

Reply:
Could it maybe be renamed to `condition_models` instead. I find the name to be a bit confusing and it prompts me to think of a many to many association (because of the rails table name expectation when you have many to many)